Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM starefossen/ruby-node
+ENV BUNDLER_VERSION=2.1.4
+
+RUN apt-get update -qq && apt-get install -y
+RUN apt-get -y install cups-client
```

Comment:
Think we might be able to remove cups-client here as it is now in psd-deployment?